.github/workflows/release.yml

```diff
@@ -0,0 +1,174 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version to release (e.g. v1.2.3)"
+        required: true
+      kind:
+        description: "Release kind"
+        type: choice
+        options: [minor, patch]
+        required: true
+        default: "minor"
+
+jobs:
+  verify-inputs:
+    name: Verify inputs
+    runs-on: ubuntu-22.04
+    env:
+      FULL_VERSION: ${{ inputs.version }}
+    outputs:
+      WITHOUT_V: ${{ steps.version-info.outputs.WITHOUT_V }}
+      PART_MAJOR: ${{ steps.version-info.outputs.PART_MAJOR }}
+      PART_MINOR: ${{ steps.version-info.outputs.PART_MINOR }}
+      PART_PATCH: ${{ steps.version-info.outputs.PART_PATCH }}
+      MAJOR: ${{ steps.version-info.outputs.MAJOR }}
+      MAJOR_MINOR: ${{ steps.version-info.outputs.MAJOR_MINOR }}
+      MAJOR_MINOR_PATCH: ${{ steps.version-info.outputs.MAJOR_MINOR_PATCH }}
+      RELEASE_BRANCH: ${{ steps.version-info.outputs.RELEASE_BRANCH }}
+    steps:
+      - name: Verify version
+        run: |
+          if [[ ! "${FULL_VERSION}" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            echo "Version must be in the form of vX.Y.Z"
+            exit 1
+          fi
+      - name: Extract version info
+        id: version-info
+        run: |
+          WITHOUT_V=${FULL_VERSION#v}
+          PART_MAJOR=${WITHOUT_V%%.*}
+          PART_MINOR=${WITHOUT_V#*.}
+          PART_MINOR=${PART_MINOR%%.*}
+          PART_PATCH=${WITHOUT_V##*.}
+          {
+            echo "WITHOUT_V=${WITHOUT_V}"
+            echo "PART_MAJOR=${PART_MAJOR}"
+            echo "PART_MINOR=${PART_MINOR}"
+            echo "PART_PATCH=${PART_PATCH}"
+            echo "MAJOR=${PART_MAJOR}"
+            echo "MAJOR_MINOR=${PART_MAJOR}.${PART_MINOR}"
+            echo "MAJOR_MINOR_PATCH=${PART_MAJOR}.${PART_MINOR}.${PART_PATCH}"
+            echo "RELEASE_BRANCH=release/v${PART_MAJOR}.${PART_MINOR}"
+          } >> "$GITHUB_OUTPUT"
+
+  prepare-release-branch:
+    name: Prepare release branch
+    if: inputs.kind == 'minor'
+    runs-on: ubuntu-22.04
+    needs: verify-inputs
+    env:
+      BRANCH: ${{ needs.verify-inputs.outputs.RELEASE_BRANCH }}
+    steps:
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+        with:
+          ref: ${{ github.head_ref }}
+      - name: Create release branch
+        run: |
+          git fetch
+          git pull
+          git checkout "${BRANCH}" || git checkout -B "${BRANCH}"
+          git push origin "${BRANCH}"
+
+  micro-services:
+    needs: [verify-inputs, prepare-release-branch]
```

Comment:
Does this work if prepare-release-branch is skipped?

Reply:
https://github.com/edgelesssys/constellation/actions/runs/3895082895 :disappointed: